src/globals/grid/classic/_classic.scss

```
@@ -64,7 +64,8 @@ $grid-gutter-breakpoints: (
 
   @for $i from 1 through $columns {
     .#{$prefix}--col-xs-#{$i} {
-      flex-basis: (100% * $i / $columns);
+      flex-basis: auto;
+      width: (100% * $i / $columns);
```

Instead of switching to auto, could we do:

```suggestion
      flex: 0 0 (100% * $i / $columns);
      max-width: (100% * $i / $columns);
```

Seems like this works for bootstrap's grid in IE11 so might be worth a shot instead of coding width directly.

---

I tried your suggestion @joshblack but it doesn't work in IE11

---

What's the layout code that you're trying out? I would think `max-width` being the same as `width` would remedy it here, but seems like from our conversation earlier that `auto` is important, potentially?

---

Oh wait, I think I only took the `flex` property and I didn't add the `max-width`. I can confirm that using your suggestion still works in IE11